cmd/replicate-edit.go

```diff
@@ -60,6 +60,10 @@ var replicateEditFlags = []cli.Flag{
 		Name:  "replicate",
 		Usage: "comma separated list to enable replication of delete markers, and/or deletion of versioned objects.Valid options are \"delete-marker\", \"delete\" and \"\"",
```

Comment:
Q: Is there a reason to not include replica-sync here, since its a modifier on replication behavior? Or do we want to keep this only for delete-related behavior (in which case, calling it `--replicate` is a bit odd)

Reply:
that's actually a good idea - we could do that

Reply:
To address the concern above w.r.t. specifying this in active-passive - if I set `--replicate "delete,delete-marker,replica-sync" in active-passive, nothing should actually happen right? Because on Site A, no object would ever have replica status. Setting it right away in active-passive should have no "visible" effect, nor would enabling it after initially starting replication.

If a user then did enable active-active, replica sync would work as expected - though they would need to set `replica-sync` on both ends. The documentation can include explicit warnings against setting the field on only one side of active-active, as you wont see true synchronization on both sides.

Reply:
in an active passive setup this setting would have no meaning - since no object would be in Replica status. The default is always replica-sync enabled. So in an active-active scenario, if they edit a rule and don't set the replica-metadata-sync while changing delete or delete-marker replication setting, it would essentially disable it. The docs need to have a caveat on that.